### PR TITLE
Converted search_with_query_task to 4.0

### PR DESCRIPTION
**Pull request by alaframboise**

### CODE REVIEW

File: develop/jsapi/search_with_query_task/index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              addFeatures(results.features);
+            })
+          } else {
+            addFeatures(results.features);
+          }
+        });
+      }
+
+      // Add features as graphics
+      function addFeatures(features) {
```
Comment (nixta) [inline develop/jsapi/search_with_query_task/index.html]: Why not set a unique value renderer on the layer?

File: develop/jsapi/search_with_query_task/index.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          outline: {
+            color: [ 255, 255, 255 ],
+            width: 1
+          }
+        });
+        // Set symbol and popup template
+        for (var i=0; i < features.length; i++) {
+          var feature = features[i];
+          feature.symbol = symbol;
+          feature.popupTemplate = {
```
Comment (nixta) [inline develop/jsapi/search_with_query_task/index.html]: Likewise an infoTemplate?

File: develop/jsapi/search_with_query_task/index.html (hunk 1 above)
Comment (alaframboise) [inline develop/jsapi/search_with_query_task/index.html]: I thought about that but it's more code. I was thinking that would go in a different lab.

File: develop/jsapi/search_with_query_task/index.html (hunk 2 above)
Comment (alaframboise) [inline develop/jsapi/search_with_query_task/index.html]: The problem is that the amount of code is already distracting enough from the focus on query task.

Comment (alaframboise) [inline develop/jsapi/search_with_query_task/index.html]: A better question is: Why is feature.symbol null? And why doesn't the feature come back with a proper symbol given that the renderer is stored with this layer..., but that's another story.

**Pull request merged**